stories/theme/ag/large/Modals/MoreOptionsModal.css

```diff
@@ -0,0 +1,199 @@
+@value HeaderHeight 50px;
+@value MoreOptionsModalHeight calc(100vh - HeaderHeight - 10px);
+@value MoreOptionsWidth 1024px;
+
+.MoreOptionsModalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  justify-content: center;
+  z-index: 10;
+  display: inline-block;
+  align-items: center;
+  -webkit-box-pack: center;
```

Comment:
The autoprefixer adds this so this isn’t needed.

Reply:
fixed